src/AaveV2EthRiskParams_20230702/AaveV2EthRiskParams_20230702.sol

```diff
@@ -0,0 +1,213 @@
+  uint256 public constant BAL_LIQUIDATION_THRESHOLD = 55_00; // 70 -> 55
+  uint256 public constant BAL_LIQUIDATION_BONUS = 10800; // unchanged
+  uint256 public constant BAL_RF = 30_00; // 20 -> 30
+
+  uint256 public constant BAT_LTV = 0; // 65 -> 0
+  uint256 public constant BAT_LIQUIDATION_THRESHOLD = 52_00; // 70 -> 52
+  uint256 public constant BAT_LIQUIDATION_BONUS = 11000; // 7.5 -> 10
+  uint256 public constant BAT_RF = 30_00; // 20 -> 30
+
+  uint256 public constant CVX_LTV = 0; // 35 -> 0
+  uint256 public constant CVX_LIQUIDATION_THRESHOLD = 40_00; // 45 -> 40
+  uint256 public constant CVX_LIQUIDATION_BONUS = 10850; // unchanged
+  uint256 public constant CVX_RF = 30_00; // 20 -> 30
+
+  uint256 public constant DPI_LTV = 0; // 65 -> 0
+  uint256 public constant DPI_LIQUIDATION_THRESHOLD = 42_00; // 70 -> 42
+  uint256 public constant DPI_LIQUIDATION_BONUS = 11000; // 7.5 -> 10
+  uint256 public constant DPI_RF = 30_00; // 20 -> 30
+
+  uint256 public constant ENJ_LTV = 0; // 60 -> 0
+  uint256 public constant ENJ_LIQUIDATION_THRESHOLD = 60_00; // 67 -> 60
+  uint256 public constant ENJ_LIQUIDATION_BONUS = 11000; // 6 -> 10
+  uint256 public constant ENJ_RF = 30_00; // 20 -> 30
+
+  uint256 public constant KNC_LTV = 0; // 60 -> 0
+  uint256 public constant KNC_LIQUIDATION_THRESHOLD = 1_00; // 70 -> 1
+  uint256 public constant KNC_LIQUIDATION_BONUS = 11000; // unchanged
+  uint256 public constant KNC_RF = 30_00; // 20 -> 30
+
+  uint256 public constant MANA_LTV = 0; // 61.5 -> 0
+  uint256 public constant MANA_LIQUIDATION_THRESHOLD = 62_00; // 75 -> 62
+  uint256 public constant MANA_LIQUIDATION_BONUS = 11000; // 7.5 -> 10
+  uint256 public constant MANA_RF = 45_00; // 35 -> 45
+
+  uint256 public constant REN_LTV = 0; // 55 -> 0
+  uint256 public constant REN_LIQUIDATION_THRESHOLD = 40_00; // 60 -> 40
+  uint256 public constant REN_LIQUIDATION_BONUS = 11000; // 7.5 -> 10
+  uint256 public constant REN_RF = 30_00; // 20 -> 30
+
+  uint256 public constant xSUSHI_LTV = 0; // 50 -> 0
+  uint256 public constant xSUSHI_LIQUIDATION_THRESHOLD = 60_00; // 65 -> 60
+  uint256 public constant xSUSHI_LIQUIDATION_BONUS = 11000; // 8.5 -> 10
+  uint256 public constant xSUSHI_RF = 45_00; // 35 -> 45
+
+  uint256 public constant YFI_LTV = 0; // 50 -> 0
+  uint256 public constant YFI_LIQUIDATION_THRESHOLD = 55_00; // 65 -> 55
+  uint256 public constant YFI_LIQUIDATION_BONUS = 11000; // 7.5 -> 10
+  uint256 public constant YFI_RF = 30_00; // 20 -> 30
+
+  uint256 public constant ZRX_LTV = 0; // 55 -> 0
+  uint256 public constant ZRX_LIQUIDATION_THRESHOLD = 45_00; // 65 -> 45
+  uint256 public constant ZRX_LIQUIDATION_BONUS = 11000; // 7.5 -> 10
+  uint256 public constant ZRX_RF = 30_00; // 20 -> 30
+
+  function _postExecute() internal override {
+    // BAL
+    ILendingPoolConfigurator(AaveV2Ethereum.POOL_CONFIGURATOR).setReserveFactor(
+      AaveV2EthereumAssets.BAL_UNDERLYING,
+      BAL_RF
+    );
+
```

Comment:
maybe remove the space between setReserveFactor  and configureReserveAsCollateral so each asset is block

Reply:
removed